tox.ini

```diff
@@ -39,7 +38,6 @@ commands =
     tests: hdev config --if tool.hdev.tox.test_extras --run --template "pip install --quiet --use-feature=in-tree-build \{0\}"
     tests: coverage run -m pytest tests
 
-    lint: mypy src/h_assets
```

Comment:
This is just causing trouble, and also means this project diverges from the template.